client/components/forms/multi-checkbox/index.jsx

```diff
@@ -43,24 +37,27 @@ var MultiCheckbox = module.exports = React.createClass({
 		} );
 
 		event.stopPropagation();
-	},
-
-	getCheckboxElements: function() {
-		var checked = this.props.checked || this.state.initialChecked;
-
-		return this.props.options.map( function( option ) {
-			var isChecked = checked.indexOf( option.value ) !== -1;
-
-			return (
-				<label key={ option.value }>
-					<input name={ this.props.name + '[]' } type="checkbox" value={ option.value } checked={ isChecked } onChange={ this.handleChange } disabled={ this.props.disabled } />
-					<span>{ option.label }</span>
-				</label>
-			);
-		}, this );
-	},
-
-	render: function() {
-		return <div className="form-checkbox-group" { ...omit( this.props, Object.keys( MultiCheckbox.propTypes ) ) }>{ this.getCheckboxElements() }</div>;
+	};
+
+	render() {
+		const { disabled, name, options } = this.props;
+		const checked = this.props.checked || this.state.initialChecked;
+		return (
+			<div className="multi-checkbox" { ...omit( this.props, Object.keys( MultiCheckbox.propTypes ) ) }>
```

Comment:
As an aside and acknowledging it was already here previously, the pattern of `omit` by `propTypes` is something I hope we can move away from, since it prevents us from using this useful Babel transform in production:

https://github.com/oliviertassinari/babel-plugin-transform-react-remove-prop-types